boot/reboot.go

```diff
@@ -57,9 +57,22 @@ var (
 	shutdownMsg = i18n.G("reboot scheduled to update the system")
 	haltMsg     = i18n.G("system halt scheduled")
 	poweroffMsg = i18n.G("system poweroff scheduled")
+
+	// testingRebootItself is set to true when we want to unit
+	// test the Reboot function.
+	testingRebootItself = false
 )
 
+func EnableTestingRebootFunction() (restore func()) {
```

Comment:
we shouldn't want to do this from any other package, so I think this belongs to export_test.go, no?

Reply:
Changed now